torch/distributed/fsdp/_utils.py

```diff
@@ -78,3 +83,51 @@ def f(module: torch.nn.Module, prefix: str, *args, **kwargs):
 
     f(root_module, "", *args, **kwargs)
     return return_fn(*args, **kwargs)
+
+
+@torch.no_grad()
+def _alloc_storage(tensor: torch.Tensor, size: torch.Size) -> bool:
```

Comment:
I originally used the `bool` return value, but I no longer do. I kept them for now since they might be useful in the future, but I can remove them if you want.